.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "7"
```

Comment:
according to https://github.com/nodejs/Release#release-schedule, Node 7 has been EOL'ed. current stable is Node 9 and LTS is Node 8. maybe just put "node" here to specify the latest stable release? (according to https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions)

Reply:
+1 for `node`

Reply:
Done